msteams-platform/concepts/authentication/nested-authentication.md

```diff
@@ -0,0 +1,212 @@
+| Acquire access token through MSAL.js |  | ✔️ * Requires new msalconfig property: supportsNestedAppAuth |
+| Solicit user consent for more permissions | ✔️ |  |
+| Conduct an OBO exchange on the server | ✔️ |  |
+
+## Scenarios
+
+| Scenario | description |
+| --- | --- |
+| **Consenting to SSO (and other permissions)** | Tom, a new member of the Contoso design team, needs to use Mural in Teams to collaborate on whiteboards. Upon first use, a dialog prompts Tom to grant permissions, including reading their profile for their avatar (User.Read). After tom consents, Tom can use Mural seamlessly in future meetings across devices. |
+| **Reauthentication or Conditional Access step-up auth** |  Tom, while working from Australia, encounters a conditional access trigger requiring multifactor authentication (MFA) to access Mural in Teams. A dialog informs Tom of the more verification needed, leading them through the MFA process to continue using Mural. |
+| **Errors** | Tom faces an error with Mural displaying an 'Oh no!' page due to an issue retrieving account information. A retry button prompts Tom to reauthenticate, only to find that the system administrator blocked access to Mural. |
+
+## Configure nested authentication
+
+> [!NOTE]
+> Nested authentication is in developer preview and not supported by all host environments. Developers need to check the support status using the Teams JS SDK and provide a fallback experience for unsupported environments.
+
+To use nested authentication, follow these steps:
+
+1. [Register their single page application](#register-your-single-page-application)
+1. [Add trusted brokers](#add-trusted-brokers)
+1. [Enable the native bridging](#enable-native-bridging)
+1. [Initialize public client application](#initialize-public-client-application)
+1. [Call an API](#call-an-api)
+
+### Register your single-page application
+
+You need to create a Microsoft Azure App registration for your add-in on the Azure portal. The app registration must have a name, supported account type, and SPA redirect. If your add-in requires additional app registration beyond NAA and SSO, see [Register your single-page application.](/entra/identity-platform/scenario-spa-app-registration)
+
+### Add trusted brokers
+
+To enable nested authentication, your application must actively configure one or more specialized redirect URIs to signal to the Microsoft identity platform that it's allowed to be brokered by supported hosts. The redirect URI of the application must be of type **Single Page Application** and conform to the following scheme:
+
+```
+brk-<broker_application_id>://<your_domain>
+```
+
+Where <broker_application_id> is the app ID or alias of the broker or brokers you wish to trust and <your_domain> is the fully qualified domain name where your application is hosted. For example, **brk-multihub://contoso.com**.
+
+For Teams only apps, add two Redirect URIs to indicate you trust only Teams to broker tokens for you:
```

Comment:
We have decided not to give the Teams only option to 3P developers. So the only guidance here should be to use the brk-multihub scheme

Reply:
updated to only use brk-multihub://<your_domain>.